meson.build

```diff
@@ -0,0 +1,45 @@
+project('pyamg', 'cpp',
```

Comment:
So, meson only installs the amg_core stuff here?

Reply:
this describes everything, not just `amg_core`.  (see the `install_subdir` commmand below)

Reply:
Oh, I see.  It's such a short command, but `install_subdir` takes care of the rest (non amg_core stuff).